stockflux-chart/src/index.js

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import '../node_modules/malihu-custom-scrollbar-plugin/jquery.mCustomScrollbar.css';
```

Comment:
Do you need this custom scrollbar behaviour?

Reply:
Bin it, speak to @oriondean about his approach in the watch list.